integration_tests/src/main/python/benchmark.py

```diff
@@ -89,6 +89,10 @@ def main():
                         help='The number of iterations to run (defaults to 1)')
     parser.add_argument('--gc-between-runs', required=False, action='store_true',
                         help='Whether to call System.gc between iterations')
+    parser.add_argument('--upload-uri', required=False,
+                        help='Upload URI for summary output')
+    parser.add_argument('--upload-path', required=False,
+                        help='Upload path for summary output')
```

Comment:
Why do we need to specify the URI and path separately?  It will be very confusing/surprising if they specify a path without a URI and it silently does nothing.

IMO the URI should be the path.  For example, `hdfs://somenamenode/path/to/where/reports/go` or `s3://somebucket/path/to/reports` or `file:/some/local/reports/dir`.  I'm not seeing the benefit of having a URI and a path when a URI can be a fully qualified path.

Reply:
Thanks @jlowe I have made that change and tested it out locally. That is definitely simpler.